ExternalScripts(DONOTINCLUDEINRELEASE)/scraper/item_drop/item_drop_formatter.py

```
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+json_path = 'item_drop/item_drop_data.json' #input
+lua_path = 'item_drop/item_drop_data.lua' #output
+
+class ItemDropFormatter:
+
+    def __call__(self, **kwargs):
+        self.__format()
+
+    def __format(self) -> None:
+        npc_input = self.__load_json_file(json_path)
```

Ruff (0.14.13)

Local variable `npc_input` is assigned to but never used

Remove assignment to unused variable `npc_input`

(F841)